ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -115,6 +115,10 @@
   DECODE_ERROR(-50100, "Unable to decode the private signed raw transaction"),
   GET_PRIVATE_TRANSACTION_NONCE_ERROR(-50100, "Unable to determine nonce for account in group."),
 
+  // Privacy multi-tenancy errors
+  PRIVACY_MULTI_TENANCY_NO_TOKEN(-50100, "No token provided"),
```

Comment:
nit: Is this actually just a http "unauthorised" response? Or is that too vague?
If this is the right way to go ... is "token" the right word" Or do we want to say something else?

Reply:
done